staging/src/k8s.io/cluster-bootstrap/token/util/helpers.go

```diff
@@ -22,9 +22,12 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+	"time"
 
+	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/cluster-bootstrap/token/api"
+	"k8s.io/klog"
 )
```

Comment:
file comment: util/helpers.go as a name is terrible x2. Multiple small packages are preferred. Maybe don't move the existing since that'd break consumers, but you can add new ones to sub-packages?